### PR TITLE
Csv import/dbcreation bug

### PR DESCRIPTION
**Pull request by sloypog**
### CATEGORY

Choose one

- [x] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/config.py [hunk 1]
```diff
@@ -384,8 +384,8 @@ def _try_json_readfile(filepath):
 
 # Set this API key to enable Mapbox visualizations
 MAPBOX_API_KEY = os.environ.get("MAPBOX_API_KEY", "")
-POSTGRES_USERNAME = os.environ.get("POSTGRES_USERNAME", "")
-POSTGRES_PASSWORD = os.environ.get("POSTGRES_PASSWORD", "")
+POSTGRES_USERNAME = "postgres"
```
Comment (Sascha-Gschwind) [inline superset/config.py]: Muss vom Environment geladen werden

File: superset/config.py [hunk 2]
```diff
@@ -384,8 +384,8 @@ def _try_json_readfile(filepath):
...
+POSTGRES_PASSWORD = "postgres"
```
Comment (Sascha-Gschwind) [inline superset/config.py]: Muss vom Environment geladen werden

File: superset/views/csv_import.py [hunk 3]
```diff
@@ -211,7 +218,23 @@ def _create_database(self, db_name: str, db_flavor="sqlite"):
                 + "@localhost/"
                 + db_name
             )
-            engine = sqlalchemy.create_engine(url)
+            # TODO if there is time look for a better method to do this, otherwise document it
```
Comment (Sascha-Gschwind) [inline superset/views/csv_import.py]: Don't check in TODO's in master, create an issue and point to this line of code in the issue

**Review by Sascha-Gschwind — CHANGES_REQUESTED** [review]
Please fix the comments first

**Review by Sascha-Gschwind — COMMENTED** [review]
We should probably merge the branch csv-import into this after #59 is merged and reapply the changes there

**Review by Sascha-Gschwind — APPROVED** [review]

**Review by michellekunz — APPROVED** [review]

**Pull request merged**